src/CheckoutSdk/Sessions/Channel/AppSession.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Checkout.Sessions.Channel
+{
+    public class AppSession : ChannelData
```

Comment:
I don't think that the JsonProperty declarations on this class are needed, by default the serialization can handle camel case to lower case and underscore.

Reply:
I had to add JsonProperties because `SdkEphemeralPublicKey`  is being translated to `sdk_ephem_pub_key`. So I added all just in caase

Reply:
Sure